TESTING.md

```diff
@@ -1,350 +1,352 @@
+
+**4.12** The IDE restarts.
+
+**4.13** ASSERTION: No notification about Scala plugin is shown anymore.
+
+**4.14** ASSERTION: Scala plugin is enabled.
+- Navigate to **File > Settings... > Plugins > Installed** to see this.
+
+### 5 About window
+
+**5.1** From the main menu, choose **A+ > About A+ Plugin**.
+
+**5.2** The about window is shown.
+
+**5.3** Close the window by clicking **OK**.
+
+### 6 Turn project into A+ project
+
+**6.1** From the main menu, choose **A+ > Turn project into A+ course project**.
+
+**6.2** On the dialog that opens, click **Cancel**.
+
+**6.3** ASSERTION: The dialog closes but nothing else seems to happen.
+
+**6.4** Open **A+ > Turn project into A+ course project** again.
+
+**6.5** ASSERTION: **Leave IntelliJ settings unchanged** checkbutton is not checked.
+        **Restart IntelliJ to reload settings** checkbutton is checked.
+
+**6.6** Check "leave unchanged" checkbox.
+
+**6.7** ASSERTION: "Restart" checkbox gets unchecked and disabled.
+
+**6.8** Uncheck "leave unchanged" checkbox.
+
+**6.9** ASSERTION: "Restart" checkbox gets enabled.
+
+**6.10** Check "leave unchanged" checkbox again and click OK.
+
+**6.11** ASSERTION: **O1Library** appears as a module in the project tree.
+
+**6.12** ASSERTION: **Modules** tool window shows a list of O1 modules.
+
+**6.13** Once again, navigate to **A+ > Turn project into A+ course project**.
+
+**6.14** Leave the checkboxes as they are and click **OK**.
+
+**6.15** The screen shows a dialog that tells the IDE will be restarted.
+
+**6.16** Click **OK**.
+
+**6.17** The IDE restarts.
+
+**6.18** ASSERTION: The theme has changed to dark.
+
+### 7 Importing modules
+
+**7.1** On **Modules** tool window, double-click **RobotTribes**.
+
+**7.2** ASSERTION: **RobotTribes** and **Robots** appear as modules in the project tree.
+
+**7.3** ASSERTION: **RobotTribes** and **Robots** are marked **Installed** in the modules list.
+
+**7.4** On **Modules** tool window, right-click **GoodStuff**.
+
+**7.5** On the context menu that appears, choose **Import A+ module**.
+
+**7.6** ASSERTION: **GoodStuff** appears as a module in the project tree.
+
+**7.7** ASSERTION: **GoodStuff** is marked **Installed** in the modules list.
+
+**7.8** On **Modules** tool window, select multiple uninstalled modules.
+- Hold **Ctrl**/**Cmd** key down while clicking to select many items.
+
+**7.9** Click the button with download icon
+        on the top of the **Modules** tool window
+        to install all selected modules.
+
+**7.10** ASSERTION: The chosen modules appear in the project tree.
+
+**7.11** ASSERTION: The chosen modules are marked **Installed** in the modules list.
+
+**7.12** ASSERTION: All the installed modules have their dependencies correctly configured.
+- To check this, navigate to **File > Project Structure... > Modules**
+  and make sure none of the modules is underlined in red.
+- Exit **Project Structure** window by clicking **Cancel**.
+
+### 8 Using Scala REPL
+
```

Comment:
Some of the additions made to the Scala REPL behavior (i.e. open file in editor or no module selected) should probably be included here, but that's on me, I'll perhaps add them later in some separate PR.